app/models/manageiq/providers/foreman/provider.rb

```diff
@@ -25,13 +25,66 @@ class ManageIQ::Providers::Foreman::Provider < ::Provider
   validates :name, :presence => true, :uniqueness => true
   validates :url,  :presence => true
 
+  def self.params_for_create
+    @params_for_create ||= {
+      :title  => "Configure Foreman",
```

Comment:
So, downstream we rename this to Satellite via monkey patching the description method, so this needs to use description, I think.

Reply:
Okay there's no description on the provider class so I'll add one, but we'll have to update the downstream monkeypatching to include the provider

Reply:
Done